app/forms/PersonalDetailsForm.scala

```diff
@@ -58,21 +59,32 @@ object GeneralDetailsForm {
         postCode -> of(postCodeFormatter),
         country -> of(countryFormatter),
         phone -> of(phoneNumberFormatter),
-        FastPassForm.formQualifier -> of(fastPassFormFormatter(ignoreFastPassValidations))
+        FastPassForm.formQualifier -> of(fastPassFormFormatter(ignoreFastPassValidations)),
+        edipCompleted -> of(Mappings.mayBeOptionalString("error.needsEdipCompleted.required", 31, isSdipAndNotSubmitted))
```

Comment:
Looks like this validation gets triggered once the application status is not in submitted status. I hope we want to just trigger this validation only for sdip applications which are created and inprogress.